scipy/interpolate/tests/test_interpolate.py

```diff
@@ -1341,6 +1341,12 @@ def test_yi_trailing_dims(self):
         pp = BPoly.from_derivatives(xi, yi)
         assert_equal(pp.c.shape, (2*k, m, 6, 7, 8))
 
+    def test_gh_5430(self):
+        orders = np.int64(1)
```

Comment:
It very well might be python version issue --- either way, the test needs to test for both :-).